garrysmod/lua/includes/extensions/util.lua

```diff
@@ -347,3 +347,64 @@ function util.RemovePData( steamid, name )
 	sql.Query( "DELETE FROM playerpdata WHERE infoid = "..SQLStr(name) )
 	
 end
+
+--[[---------------------------------------------------------
+   Name: ClipTraceToPlayers
+   Desc: Traces across players and returns a modified trace
+-----------------------------------------------------------]]
+local function DistanceToTrace( pos, traceStart, traceEnd, along, pointOnTrace )
+	local to = pos - traceStart
+	local dir = traceEnd - traceStart
+	//Dunno if NormalizeInPlace translates to GetNormalized or Normalize

```

Comment:
FYI, NormalizeInPlace means that it directly mutates the given vector (in place). Looking at the documentation for both GetNormalized and Normalize, GetNormalized returns a new vector while leaving the source unmutated. Normalize mutates the original vector. Thus, Normalize would be the correct translation for NormalizeInPlace. This would mean that there wouldn't be a return value, so you'd have to be aware of that.